examples/parallel_orbit_5k.py

```diff
@@ -0,0 +1,113 @@
+from typing import Tuple
+from dataclasses import dataclass
+import argparse
+import functools
+
+import torch
+import torch.nn as nn
+from torch.optim import Adam
+from torch.distributed.fsdp.wrap import transformer_auto_wrap_policy
+
+from gdeep.search.hpo import GiottoSummaryWriter
+from gdeep.topology_layers import PersformerWrapper, persformer_block
+from gdeep.topology_layers.persformer_config import PoolerType
+from gdeep.trainer.trainer import Trainer, Parallelism
+from gdeep.data.datasets import OrbitsGenerator, DataLoaderKwargs
+import gdeep.utility_examples.args
+
+def main(args):
+    # Generate a configuration file with the parameters of the desired dataset
+    @dataclass
+    class Orbit5kConfig():
+        batch_size_train: int = args.batch_size
+        num_orbits_per_class: int = 1000
+        validation_percentage: float = 0.0
+        test_percentage: float = 0.0
+        num_jobs: int = 8
+        dynamical_system: str = "classical_convention"
+        homology_dimensions: Tuple[int, int] = (0, 1)  # type: ignore
```

Comment:
Why `type: ignore` here?

Reply:
Probably a mistake. It will be fixed in the next push.